src/MessageItem.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import styled from 'styled-components';
+import UserInfo from './UserInfo.json'
+
+type StyledProps = {
+    background: string;
+}
+
+type PositionProps = {
+    position?: string;
+}
+
+const StyledPosition = styled.div<PositionProps>`
+    ${(props) => 
+        props.position === "right" ?
+        `
+        margin-left: auto;
+        `
+        :
+        `
+        margin-right: auto;
+        `
+    }
+`
+
+const StyledMsg = styled.div<StyledProps>`
+    background: ${(props) => props.background};
+    display:inline-block;
+    margin: 3px;
+    padding: 3px;
+    border: 25px;
+    max-width: 220px;
+
+`
```

Comment:
```suggestion
const StyledMsg = styled.div<StyledProps>`
    background: ${({background}) => background};
    display:inline-block;
    margin: 3px;
    padding: 3px;
    border: 25px;
    max-width: 220px;

`
```

여기에도 구조분해할당을 적용할 수 있어요!!